src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -210,11 +235,19 @@ export class LdPagination {
           class={getClassNames([
             'ld-pagination',
             this.size && `ld-pagination--${this.size}`,
+            isDots && `ld-pagination--dots`,
+            this.brandColor && 'ld-pagination--brand-color',
           ])}
           part="wrapper"
+          style={styleDots}
         >
           {!this.hideStartEnd && (
-            <li class="ld-pagination__arrow">
+            <li
+              class={getClassNames([
+                'ld-pagination__arrow',
+                isDots && `ld-pagination__arrow--left`,
+              ])}
+            >
```

Comment:
```suggestion
            <li class="ld-pagination__arrow">
```

Similar for the occurrences below.